kvmagent/kvmagent/test/unittest_tools/prepare_env.sh

```
@@ -75,67 +75,64 @@ prepare_internal_repo
-    cp pip.conf ~/.config/pip/ || true
-    cp pip.conf ~/.pip/ || true
-
+    pip3.11 config set global.index-url file:///root/.zguest/zstack-utility/zstackbuild/pypi_source/pypi/simple
+    pip3.11 config set global.extra-index-url file:///root/.zguest/zstack-utility/kvmagent/kvmagent/test/unittest_tools/unittest_pypi_source/pypi/simple
 
-    # wget -c http://minio.zstack.io:9001/download/prsystem/UtilityUT/get-pip.py -O get-pip.py
     cd /root/.zguest/zstack-utility/kvmagent/kvmagent/test/unittest_tools/unittest_pypi_source/
-    pip install -r requirements/requirements1.txt -i file://`pwd`/pypi/simple
-    pip install -r requirements/requirements2.txt -i file://`pwd`/pypi/simple
-    virtualenv --version || pip install virtualenv==12.1.1
+    pip3.11 install -r requirements/requirements1.txt -i file://`pwd`/pypi/simple
```

Shellcheck (0.11.0)

[warning] Quote this to prevent word splitting.

(SC2046)